karspexet/ticket/migrations/0011_reservation_reservation_code.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2017-11-21 19:16
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ticket', '0010_unique_ticket_per_seat_and_show_20170911_0554'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='reservation',
+            name='reservation_code',
+            field=models.CharField(default='', max_length=16, unique=True),
```

Comment:
I think this might need updating too. If not, then a RunPython migration to set generated default values instead.

Reply:
Ah, yes. Good point. I'll do that before I merge this :)